src/app/pages/IdxPage/index.jsx

```diff
@@ -1,23 +1,35 @@
 import React, { Fragment } from 'react';
 import path from 'ramda/src/path';
 import PageContainer from '#lib/pageStyles/PageContainer';
+import MetadataContainer from '#containers/Metadata';
 import FrontPageSection from '#containers/FrontPageSection';
 import idxPageDataPropTypes from '#models/propTypes/idxPage';
 
 const IdxPage = ({ pageData }) => {
   const groups = path(['content', 'groups'], pageData);
+  const summary = path(['metadata', 'summary'], pageData);
+  const title = path(['metadata', 'title'], pageData);
+  const lang = path(['metadata', 'language'], pageData);
 
   return (
-    <main role="main">
-      <h1 id="content">IDX Page</h1>
-      <PageContainer>
-        {groups.map((group, index) => (
-          <Fragment key={group.title}>
-            <FrontPageSection group={group} sectionNumber={index} />
-          </Fragment>
-        ))}
-      </PageContainer>
-    </main>
+    <>
+      <MetadataContainer
+        title={title}
+        lang={lang}
+        description={summary}
+        openGraphType="website"
+      />
```

Comment:
The metadata also includes LinkedData. We should ensure we have the same linked data for these IDX pages as for the front page. 
```
const seoTitle = path(['promo', 'name'], pageData);
```


```suggestion
      />
      <LinkedData type="WebPage" seoTitle={seoTitle} />
```

Reply:
Thanks @sareh, for pointing that out.

Reply:
There was a separate issue for that https://github.com/bbc/simorgh/issues/6621. Can you link it to this PR and let Ruth know, please?

Reply:
Oh, I had no idea there was an issue for that, thank you Denis, I'll inform Ruth.